src/test/utils.test.ts

```diff
@@ -286,4 +286,42 @@ describe('Utils', () => {
 			should.equal('key @ name', utils.generateReferenceValueForLabel(undefined, new ConfigMapRef(new KeyRef('name', 'key'))));
 		});
 	});
+
+	context('Load/Save of a Secrets file', () => {
+
+		let dummyNonSecretFile: string;
+
+		before( () => {
+			dummyNonSecretFile = path.resolve(__dirname, '../../testFixture', `dummy-secret.yaml`);
+			fs.writeFileSync(dummyNonSecretFile, 'test');
+		});
+
+		after( () => {
+			fs.unlinkSync(dummyNonSecretFile);
+		});
+
+		it('should match the secrets model contents between a save and reload to/from a secret file', async() => {
+			const name: string = 'mysecret';
+			const fpOrig: string = path.resolve(__dirname, '../../testFixture', `${name}.yaml`);
+			const fpTest: string = path.resolve(__dirname, '../../testFixture', `${name}2.yaml`);
+			const yamlDoc:SecretConfig = await utils.loadSecretsFromFile(fpOrig);
+			should.exist(yamlDoc);
+			await utils.saveSecretsToFile(yamlDoc, fpTest);
+			should.exist(utils.validateFileNotExisting(name));
```

Comment:
This test may work fine, but the line is a bit confusing. So... it should exist, but we're validating that the file doesn't exist?

Reply:
`should.exist` validates a value to be not `undefined`
`utils.validateFileNotExisting(name)` will return an error string whenever the file exists.
So its basically just a check that the fresh saved file is existing on file system. 
its basically `utils.validateFileNotExisting(name) !== undefined`